src/D2L.CodeStyle.Analyzers/Language/RequireNamedArgumentsAnalyzer.cs

```diff
@@ -154,6 +156,37 @@ ArgumentListSyntax args
 			}
 		}
 
+		private static string GetPsuedoName( ArgumentSyntax arg ) {
+			string ident = null;
+
+			switch( arg.Expression ) {
+				case IdentifierNameSyntax identArg:
+					ident = identArg.Identifier.ValueText;
+					break;
+				case MemberAccessExpressionSyntax access:
+					// Member access is left-associative, so we pick the
+					// right -most ident, i.e. "foo.bar.baz" is equivalent to
+					// (foo.bar).baz, and we will grab "baz".
+					ident = access.Name.Identifier.ValueText;
+					break;
+			}
+
+			if( ident == null ) {
+				return null;
+			}
+
+			// Strip uninteresting prefixes off the identifier
+			if ( ident.StartsWith( "m_" ) || ident.StartsWith( "s_" ) ) {

```

Comment:
We do `s_something` in the LMS..? I've never seen that..

Reply:
It is uncommon but does happen. It happens more in some of the libraries (which also use these analyzers.)

Reply:
I personally don't do it. I see a bit of an argument for it but *shrug*.